department-of-veterans-affairs/caseflow
# Fix for Document type filter restricting itself

Resolves [Document type filter restricting itself](https://jira.devops.va.gov/browse/APPEALS-30174)

# Description
Previously the Document type filter would break for users in the reader portion of Caseflow. These changes move the list of documents from the frozenDocTypes state to the redux store so that it may persist across pages.
## Acceptance Criteria
- [Z] Code compiles correctly

## Testing Plan

1. Log into caseflow with a user who has reader access (BVAAABSHIRE or BVADWISE)
2. Navigate to reader
3. Click the documents filter dropdown, choose 1 or more filters (but not all of them)
4. Open a document by clicking its blue link inside the table
5. Close the document so that reader is visible again.
6. Click the documents filter dropdown and note that all the options are there instead of only the ones selected.

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="1365" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/0e4192cd-e587-4cde-bd77-1545ccdc948b">
|
<img width="1376" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/a9a81961-fe8e-4b7d-a3e8-e4098e01f25e">
